openmrs/src/main/resources/webapp/messages/messages.properties

```diff
@@ -92,6 +92,12 @@ openMRS.address.startDate=Start Date
 openMRS.address.endDate=End Date
 openMRS.address.fullAddress=Full Address
 
+#Visit
+openMRS.visit.uuid=Visit UUID
+openMRS.visit.startDatetime=Start datetime
+openMRS.visit.endDatetime=End datetime
```

Comment:
please use in code stopDateTime everywhere, only here in message properties should be 'end datetime'. It will be more readable